core/src/main/java/org/stellar/anchor/sep24/Sep24Service.java

```diff
@@ -268,7 +268,8 @@ public InteractiveTransactionResponse withdraw(
     InteractiveTransactionResponse response =
         new InteractiveTransactionResponse(
             "interactive_customer_info_needed",
-            interactiveUrlConstructor.construct(txn, withdrawRequest, asset, token.getHomeDomain()),
+            interactiveUrlConstructor.construct(
+                txn, withdrawRequest, asset, token.getHomeDomain(), token),
```

Comment:
Do we need to pass `token.getHomeDomain()` when `token` is also passed?

Reply:
Good point